src/graphics/chunks.js

```diff
@@ -36,7 +36,7 @@ pc.extend(pc, (function () {
     };
 
 
-    shaderChunks.createShader = function(device, vsName, psName) {
+    shaderChunks.createShader = function(device, vsName, psName, useTransformFeedback) {
```

Comment:
That would require you a way to change the shader. Right now you the TF object contains an output buffer, which can be then processed by multiple shaders. You syntax would suggest creating many buffers for the sake of running a different shader...